### PR TITLE
libevent: fold in upstream patch for libressl 2.7

### PR DESCRIPTION
**Pull request by jerryyhom**
#### Description

Main issue is folding in an upstream patch for libressl 2.7 compilation.  Secondly, since libevent has no maintainer, I am willing to maintain it.

re: [#56400](https://trac.macports.org/ticket/56400)

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/libevent/Portfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ version             [lindex [split ${version} -] 0]
 categories          devel
 platforms           darwin
 license             BSD
-maintainers         nomaintainer
+maintainers         gmail.com:jerryyhom openmaintainer
```
Comment (pmetzger) [inline devel/libevent/Portfile]: This isn't the correct format any longer for this; you need to provide both your github handle and email address. 

`maintainer {@jerryyhom gmail.com:jerryyhom} openmaintainer`

or something similar.

**Comment by pmetzger** [comment]
You will also want to change the commit message bit from

`re: https://trac.macports.org/ticket/56400`

to

`closes: https://trac.macports.org/ticket/56400`

(Don't add a new commit, change the old one and force push.)

**Comment by jerryyhom** [comment]
Thanks.  As a git newbie, what is the proper way to change a commit?  For both the maintainer patch and commit message.

File: devel/libevent/Portfile (hunk 1 above)
Comment (jerryyhom) [reply devel/libevent/Portfile]: Hm, the MacPorts Guide does not indicate that both are required.  Is there an automatic notification, or can you explain why github handles are required?

Comment (mf2k) [reply devel/libevent/Portfile]: With the GitHub handle in there, then you will automatically get notified when someone else makes a pull request against your port(s). 

Yes, the guide needs to be updated.

Comment (jerryyhom) [reply devel/libevent/Portfile]: Okay, thanks for the explanation.  Probably I am in the minority of maintainers who do not regularly check github.

**Comment by jerryyhom** [comment]
According to the wiki page on [Commit Messages](https://trac.macports.org/wiki/CommitMessages), the syntax should be fine.

**Comment by pmetzger** [comment]
Seems to have worked. Thanks for the update!

**Pull request merged**